pkg/codegen/nodejs/gen.go

```diff
@@ -864,6 +875,7 @@ func (mod *modContext) genConfig(w io.Writer, variables []*schema.Property) erro
 		printComment(w, p.Comment, "", "")
 
 		configFetch := fmt.Sprintf("__config.%s(\"%s\")", getfunc, p.Name)
+		// TODO: handle ConstValues
```

Comment:
Could you file an issue for this?

Reply:
https://github.com/pulumi/pulumi/issues/4755